src/scss/typography/_code.scss

```diff
@@ -2,22 +2,44 @@
 @use "../tokens/font" as *;
 @use "../tokens/spacing" as *;
 
-.iati-code {
+pre > code {
+  background-color: $color-grey-10;
+  border: 1px solid $color-grey-20;
+  border-radius: 4px;
+  color: $color-grey-90;
+  display: block;
   font-family: $font-stack-monospace;
+  overflow-x: scroll;
+  padding: $padding-block;
+}
+
+:not(pre) > code {
   background-color: $color-grey-10;
-  color: #000;
   border: 1px solid $color-grey-20;
+  border-radius: 4px;
+  color: $color-grey-90;
+  font-family: $font-stack-monospace;
+  padding: 0.1em 0.2em;
+}
+
+.iati-code {
```

Comment:
I think that adding the `pre` and `code` default styles is good, but I would like to keep the `.iati-code` class for now as a way of applying the `code` styles for backwards compatibility in downstream products that are already using it in the form `<span class="iati-code">`.

Then we would need a new class name for the standard reference styles, maybe something like `.iati-reference` or `.iati-code--standard` or `.iati-code--reference`?

Reply:
Thanks @tillywoodfield. I've restored the `.iati-code` class and added a new `.iati-reference` class.

Reply:
Thanks! Is it possible to refactor slightly to remove duplicate code? I think ideally we could use `.iati-code`/`:not(pre) > code` interchangeably and also `.iati-code--block`/`pre > code` interchangeably, and not have to update the CSS in two places when we want to make updates

Reply:
Good call, didn't spot that.